HerPublicWebsite/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿using HerPublicWebsite.BusinessLogic.Models.Enums;
+
+namespace HerPublicWebsite.Extensions;
+
+public static class EnumExtensions
+{
+    public static string ForDisplay(this PropertyType propertyType)
+    {
+        return propertyType switch
+        {
+            PropertyType.Bungalow => "Bungalow",
+            PropertyType.House => "House",
+            PropertyType.ApartmentFlatOrMaisonette => "Apartment, flat, or maisonette",
+            _ => propertyType.ToString()
+        };
+    }
+    
+    public static string ForDisplay(this HouseType houseType)
+    {
+        return houseType switch
+        {
+            HouseType.EndTerrace => "End of terrace",
+            HouseType.SemiDetached => "Semi detached",
+            HouseType.Detached => "Detached",
+            HouseType.Terraced => "Terraced",
+            _ => houseType.ToString()
+        };
+    }
+    
+    public static string ForDisplay(this BungalowType bungalowType)
+    {
+        return bungalowType switch
+        {
+            BungalowType.EndTerrace => "End of terrace",
+            BungalowType.SemiDetached => "Semi detached",
+            BungalowType.Detached => "Detached",
+            BungalowType.Terraced => "Terraced",
+            _ => bungalowType.ToString()
+        };
+    }
+    
+    public static string ForDisplay(this FlatType flatType)
+    {
+        return flatType switch
+        {
+            FlatType.GroundFloor => "Ground floor",
```

Comment:
I noticed earlier that "ground floor" includes the basement, so should this be "Ground floor or basement" to avoid confusion? I think if I lived below ground I wouldn't consider myself as living "on the ground floor", so I think this would confuse me

Reply:
Oh, nice, thanks